build.gradle

```
@@ -48,11 +48,13 @@ dependencies {
 
     implementation "org.slf4j:slf4j-log4j12"
     implementation "org.slf4j:slf4j-api"
+    implementation "com.exactpro.th2:grpc-common:3.4.0"
 
     implementation "com.google.protobuf:protobuf-java-util"
 
     // Use JUnit Jupiter API for testing.
     testImplementation 'org.junit.jupiter:junit-jupiter-api:5.7.1'
+    testImplementation 'org.junit.jupiter:junit-jupiter-params:5.7.1'
 
     // Use JUnit Jupiter Engine for testing.
     testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.7.1'
```

Can be replaced with
```suggestion
    testImplementation 'org.junit.jupiter:junit-jupiter:5.7.1'
```